examples/md-flexible/TimeDiscretization.h

```diff
@@ -0,0 +1,93 @@
+//
+// Created by nicola on 13.05.19.
+//
+
+#pragma once
+#include <autopas/utils/MemoryProfiler.h>
+#include <chrono>
+#include <fstream>
+#include <iostream>
+#include "autopas/AutoPas.h"
+#include "autopas/utils/ArrayMath.h"
+
+using namespace std;
+
+template <class AutoPasTemplate>
+class TimeDiscretization {
+ public:
+  TimeDiscretization(double particleDeltaT);
+
+  virtual ~TimeDiscretization() {}
+  string arrayString(array<double, 3> input) {
```

Comment:
This functionality is already implemented in namespace `ArrayUtils` by function `to_string`